lib/src/services/jira.test.ts

```diff
@@ -4,14 +4,16 @@
 /* eslint-disable functional/no-conditional-statement */
 /* eslint-disable jest/no-conditional-expect */
 import {
-  Issue,
+  // GenericJiraIssue,
   mostRecentIssueComment,
   mostRecentIssueTransition,
   IssueChangeLog,
   issueTransitions,
-  issueLastWorked,
-  IssueWorklog,
+  // issueLastWorked,
```

Comment:
Remove commented out code.

Reply:
Removed for `jira.test.ts`. Will have to look at other files to remove other commented out code.